Tasks/AppCenterTestV1/package.json

```diff
@@ -18,7 +18,7 @@
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks",
   "dependencies": {
     "glob": "^7.0.6",
-    "appcenter-cli": "1.0.7",
+    "appcenter-cli": "^1.1.15",
```

Comment:
Do we want to relax the version here? Can we pin it to 1.1.15?

Reply:
We discussed this internally - we'd like to relax the version (it causes us problems on support when fixes don't follow through to Azure DevOps tasks) with a number of users now adding a script before the task to force the appcenter-cli to update.